.claude/skills/GenerateProperty/SKILL.md

```
@@ -0,0 +1,18 @@
+---
+name: generate-property
+description: laravelのモデルにpropertyを自動生成
+---
+
+## @propertyの自動生成
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)